app/services/languages/detection.rb

```diff
@@ -4,6 +4,10 @@ class Detection
 
     PROBABILITY_THRESHOLD = 0.5
 
+    def self.codes
```

Comment:
Can be extracted into a separate class if we need more functionality related to the language codes.